homeassistant/components/pegel_online/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: no actions/services are implemented
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
```

Comment:
I think we should also accept flows, which only have flow-descriptions (_string key `config.step.<step>.description`_) if these already explains the whole step form.
I don't think that any further (_description_) text on this forms would bring any advantage to the user-experience
| first step | second step |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/03ad9ab0-2928-453b-8697-886352cd7785) | ![image](https://github.com/user-attachments/assets/5bfd36f6-3dad-4ce7-9c42-d7408d1b5526) |

Reply:
But in this case I would also find that `(in km)` is not really relevant here as it already shows as suffix